spec/support/helpers/state_file_intake_helper.rb

```diff
@@ -95,7 +95,7 @@ def assert_flow_explorer_sample_params_includes_everything(us_state)
 
     intake = StateFile::StateInformationService.intake_class(us_state).last
     flow_explorer_params = FlowsController::SampleStateFileIntakeGenerator.send("#{us_state}_attributes")
-    intake_attribute_keys = intake.attributes.select { |_k, v| v }.keys - %w[id primary_state_id_id]
+    intake_attribute_keys = intake.attributes.select { |_k, v| v }.keys - %w[id primary_state_id_id made_az321_contributions]
```

Comment:
I don't like this but I'm not sure the alternative. This popped up because I was doing a `nested_attributes_for` for the intake, I think.